packages/visualizations/src/components/utils/MiniMap.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import tippy from './tippy';
+    import SvgChoropleth from '../Map/Svg/Choropleth.svelte';
+    import type { ColorScale } from '../types';
+    import type { ChoroplethDataValue, NavigationMap } from '../Map/types';
+
+    export let active: boolean;
+    export let showTooltip: boolean;
+    export let data: { value: ChoroplethDataValue[] };
+    export let map: NavigationMap;
+    export let colorScale: ColorScale | undefined;
+
+    $: ({ label } = map);
+    $: options = { shapes: map.shapes, colorScale };
+</script>
+
+{#if showTooltip}
+    <button
+        type="button"
+        class:active
+        on:click
+        use:tippy={{
+            content: label,
+        }}
+    >
+        <SvgChoropleth {data} {options} />
+    </button>
+{:else}
+    <button type="button" class:active on:click>
```

Comment:
`on:click` is unused ? To delete ?

Reply:
Same as above, as far as I've tested it without all this on:clik it doesn't forward the on:click passed to the component I guess